viewing_party/party.py

```diff
@@ -1,23 +1,142 @@
 # ------------- WAVE 1 --------------------
 
+
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating:
+        return {
+            "title": title,
+            "genre": genre,
+            "rating": rating
+        }
+    return None

```

Comment:
Nice work Emily! Another common pattern you'll see is the **guard clause**. A guard clause is a conditional that's used to check for falsey cases early to exit the function. This allows the primary purpose or focus of the function to be unnested and easier to read. 

```suggestion
    if not title and not genre and not rating:
        return None
   
    return {
            "title": title,
            "genre": genre,
            "rating": rating
        }
```